MaterialLib/PorousMedium/Porosity/ConstantPorosity.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   ConstantPorosity.h
+ *
+ * Created on August 16, 2016, 1:03 PM
+ */
+
+#ifndef OGS_CONSTANTPOROSITY_H
+#define OGS_CONSTANTPOROSITY_H
+
+#include "Porosity.h"
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+class ConstantPorosity final : public Porosity
+{
+public:
+
+    explicit ConstantPorosity(const double value) : _value(value)
+    {
+    }
+
+    /// Get model name.
+    std::string getName() const override
+    {
+        return "Constant porosity";
+    }
+
+    /**
```

Comment:
It would be nice if you would choose single doxygen comment style (at least per single file).